src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -948,6 +948,8 @@ private static uint GetParameterRefEscape(ParameterSymbol parameter)
             {
                 { RefKind: RefKind.None } => Binder.CurrentMethodScope,
                 { EffectiveScope: DeclarationScope.RefScoped } => Binder.CurrentMethodScope,
+                { RefKind: RefKind.Out } => Binder.ReturnOnlyScope,
```

Comment:
Is this redundant with the default case below? `_ => Binder.ReturnOnlyScope`

Reply:
It's not redundant with the default case since this `{ RefKind: RefKind.Out }` case actually covers `[UnscopedRef] out` which should be treated as _return only_ rather than _calling method_ in the next case.

Reply:
> This quote from the spec is out-of-date

https://github.com/dotnet/csharplang/pull/6688